### ISSUE TITLE
Outreach: emails to send to individuals who are dormant/inactive/etc

### ISSUE THREAD

**Issue by tellyworth** [WordPress/five-for-the-future#205]
Similar to #170, let's work out the content of emails to send to individual contributors who have pledged time but are inactive or incomplete in some way. For example:

- [ ] Pledged time to team x, hasn't logged in for 3 months or more
- [ ] Pledged time to team x, no activity on profile in last 3 months
- [ ] Pledged time to team x, recent activity on profile but none related to team x

These emails should be proactive and assume positive intent: the point is to reach out, offer help, and try to get them to engage or tell us what we're missing.

It's possible that the emails need to be somewhat different for each team. For that reason I'd suggest we start with one or two teams, and send a small number of emails initially in order to gather feedback and tweak the content.

The tasks here are:
* Write copy for these emails
* Make any decisions needed about how to send them and what to do about replies and responses

See also #194

**Comment by tellyworth** [comment]
Here's a very rough draft of an email for someone who has **Pledged time to team x, but hasn't logged in for 3 months or more**:

```
Hi NAME,

A while ago you pledged to contribute NUMBER hours a week to TEAM X of the WordPress project:

https://profiles.wordpress.org/USERNAME

We haven't seen you in a while, and we need your help!

Team X is working on some new and important projects. To help, please check out our team handbook here:

https://make.wordpress.org/TEAM-X/handbook/getting-started/

If you have any questions we'd be happy to help put you in touch with a team representative.

You can change the details of your pledge here, including selecting a different team or changing the number of hours:

https://profiles.wordpress.org/me/profile/edit/group/5/

If you have received this email in error or are no longer able to contribute to the WordPress project, you needn't take any action - we'll automatically retire your pledge in 30 days.

Thanks for your pledge, we look forward to working with you on the future of WordPress.

Regards,
The Team/Foundation/Something.
```

This could perhaps form the starting point for some specific team emails. I could imagine the Training Team might want a call-to-action for some specific projects or handbook pages for example.

**Comment by iandunn** [comment]
That looks pretty good. It might help to add something like, "if you have been contributing to {team}, but it isn't showing up on your wordpress.org profile, please let us know so we can work on crediting that in the future".

> If you have received this email in error or are no longer able to contribute to the WordPress project, you needn't take any action - we'll automatically retire your pledge in 30 days.

IIRC the plan in #169 is to only send out emails initially, and then circle back after awhile to evaluate if we need to automatically start automatically removing accounts. So we might need to tweak that a bit for the initial emails that go out in  #27

**Comment by angelasjin** [comment]
I think for people who have logged in within 3 months, let's give them a longer grace period than 30 days, as sometimes a month can just be inexplicably busy. Perhaps 60 days? 

Also, if the email is received in error and they don't take any action, I don't know that the pledge should be automatically retired. If it is received in error (for example, they contributed but for some reason it didn't show up) what would be the best way for them to notify us?

**Comment by iandunn** [comment]
In #206 I reworked the email to account for several things:

* folks pledge to multiple teams
* hardcoding links to specific handbook pages for different teams makes it a bit messy - not all teams have them, or have them at the same URL
* added a mention of Slack, since that seems like the best place to ask teams questions
* mentioned replying to the email last, to encourage talking directly to a team, and to reduce the burden on helpscout volunteers being middlemen
* removed the text about retiring pledges since that's part of a future milestone

Here's what it looks like:

<blockquote>
Hi {name}, a while ago you pledged to contribute 15 hours a week to the Accessibility, Community, Core, Design, Documentation, Hosting, Marketing, and Polyglots teams:

https://profiles.wordpress.org/{username}

We haven't seen you in a while, and we need your help! We're working on some new and important projects. If you'd like to get involved, or have any questions, you can:

* Talk to your team on our Slack workspace at https://wordpress.slack.com. Visit https://chat.wordpress.org for more information.
* Read your team's handbook by visiting https://make.wordpress.org, selecting your team, and opening the Handbook link in the menu.
* Respond to this email

If you're no longer able to contribute to the WordPress project, that's ok, we understand! Please update your pledge to reflect your current contributions, so that team representatives know who is available to help.

https://profiles.wordpress.org/me/profile/edit/group/5/

If you have been contributing, but it isn't showing up in the Activity section of your profile, please reply and let us know so we can work on crediting that in the future.

Thanks for your pledge, we look forward to working with you on the future of WordPress.

Have a great day! 
</blockquote>

We can add more complicated logic, have different templates for different teams, etc, but I'm not sure it's worth the time right now. 

I'm open to different ideas though. Any thoughts?

**Comment by angelasjin** [comment]
Awesome, thanks Ian! Is there any automated deactivation of the pledge then? 

I know you recommended against hardcoding links, but perhaps the [team rep page](https://make.wordpress.org/updates/team-reps/) would be ok, since that isn't tired to any specific team? I would add a line after your first bullet point that says: "Chat with your a team rep to learn about the team's current priorities and how you might be able to get involved"

**Comment by iandunn** [comment]
The team reps link is great, I'll add that 👍🏻 

I haven't added any automatic deactivation. My plan was to [send emails first, and then evaluate how many inactive pledges remained](https://github.com/WordPress/five-for-the-future/issues/169). If they're not effective enough, then we could add deactivation.

IIRC that was a compromise due to pushback on automatic deactivation, but IMO the chances of emails alone being effective are very low.

**Comment by naokomc** [comment]
The email content looks good, and I was thinking about what Angela mentioned – someone to reach out to in case they are not sure where to get started.

For Polyglots Team I think the current activities logged are enough. But in case there are missing activities, or if people misunderstood the pledge scope/requirements, it's safe not to auto-deactivate for this time around.

**Comment by hlashbrooke** [comment]
This wording is solid and I like the suggestion of adding the link to the team reps page.

**Comment by tobifjellner** [comment]
I was thinking about the phrase:
> We're working on some new and important projects.

For Polyglots, the goals or projects may vary wildly between various locales. Would it be useful stressing something like: "As you know, every aspect of the WordPress project is built on voluntary contributions like yours, and there's always something needed, where your contribution would be of good help."

**Comment by chaion07** [comment]
> I think for people who have logged in within 3 months, let's give them a longer grace period than 30 days, as sometimes a month can just be inexplicably busy. Perhaps 60 days?
> 
> Also, if the email is received in error and they don't take any action, I don't know that the pledge should be automatically retired. If it is received in error (for example, they contributed but for some reason it didn't show up) what would be the best way for them to notify us?

Strongly agree on the increase of the 'grace period' of more than 30 days. Can we have 90 days to accommodate more people? Thanks!

**Comment by chaion07** [comment]
Is there a way we can redirect folks who pledged for the teams to contact back on the Slack Channel for the respective teams? Can we link the WordPress Org Slack within the email content? Thanks!

**Comment by iandunn** [comment]
The email has this:

> Talk to your team on our Slack workspace at https://wordpress.slack.com/. Visit https://chat.wordpress.org/ for more information.

Does that cover what you were thinking? If not, could you give an example of what you meant?

**Comment by chaion07** [comment]
> The email has this:
> 
> > Talk to your team on our Slack workspace at https://wordpress.slack.com/. Visit https://chat.wordpress.org/ for more information.
> 
> Does that cover what you were thinking? If not, could you give an example of what you meant?

Indeed! This covers what I was looking for. Sharing another example mentioning the specific team for increased engagement:

"[Click here](https://wordpress.slack.com/archives/C02RW657Q) to connect to the Training Team on WordPress Slack. Visit https://chat.wordpress.org/ for more information.

**Comment by gidgey** [comment]
I'd like to add the following as a 0 email (prevention).

Send 30 days after pledge.

Thank you [USERNAME] for your pledge to contribute [x] hours to for Five for the Future. We believe that this program is instrumental in our 104.5% market share increase since the program began in 2014. 

We're so excited to see your contributions and can't wait to see you in Slack.

If you have any questions or would just like to jump into issues dedicated for Five to the Future, follow our GitHub board [link: https://github.com/WordPress/five-for-the-future/issues].

Thanks again,

Your Friends at WordPress.org
